styles/config/vocabularies/Technical/accept.txt

```diff
@@ -1,14 +1,17 @@
 [Aa]llowlist[s]?
 [Aa]utocommit
 [Aa]utodeploy
+[Aa]bstractive
```

Comment:
i don't think this one is generally a word we want to have in our docs. i see in the original PR it's the description for `xsum`, so we can just wrap that one line in a vale exception, i think.

Reply:
I am learning this ML space as we go. "Abstractive" is a NLP summarization method. If we have other ML-domain specific terms in the future with only one occurrence, should those also be exceptions? 
How do I add an exception in vale?

Reply:
unless they become very prevalent in our docs overall, i think exceptions are the way to go. we can always reevaluate in the future. :slightly_smiling_face: 

https://vale.sh/docs/topics/config/#markup-based-configuration

Reply:
Added [here](https://github.com/digitalocean/product-docs/pull/4268/files#diff-50da5165f231f49a31196b79bfa1bc80d4b11f54e1b93c87df2d1090d49bba4bR57).